sernet.gs.service/src/sernet/verinice/service/commands/RemoveElement.java

```diff
@@ -338,7 +340,7 @@ private boolean hasOrphanEntity(CnATreeElement element) {
     }
 
     private void deleteOrphanEntity(CnATreeElement element) {
-        if (element.getEntity() != null) {
```

Comment:
Make sure that the old delete operation remains unchanged if the following applies: `module-templates.active=false`

Please remove the next if statement from this command and implement another solution.